### DISCUSSION TITLE
Unable to make Project public

### DISCUSSION THREAD

**Discussion by Manishyadv** [community/community#31050] [Projects and Issues]
I've made a project from private to public on Github issues/projects (https://github.com/features/issues), tried accessing through URL but items are not visible. getting the following error- You can't see this item

<img width="897" alt="Screenshot 2022-08-28 at 12 10 14 AM" src="https://user-images.githubusercontent.com/27053695/187043878-b9b8ddc7-73bd-4505-9303-23198761bf43.png">

**Comment by Kirtishukla2004** [comment]
hey try this to convert a project from private to public, you can:

Right-click on the [canvas](https://github.com/kumu/docs/blob/master/overview/map-editor.html#canvas)
Select Project
Select Convert project (public/private)
Alternatively, you can:

Open the [project settings menu](https://github.com/kumu/docs/blob/master/overview/settings.html#project-settings)
Select Admin from the sidebar
Select Change privacy
**try this hope it will help**

**Reply by Manishyadv** [reply]
@shukla2004 I think we are not on the same track. My question is regarding Github projects- https://github.com/features/issues

can you please confirm, or am I still missing something?

**Comment by jfullstackdev** [comment]
try Incognito mode, that might be a cached data

**Reply by Manishyadv** [reply]
@jdevstatic not working
